experimental/traffic-portal/src/app/models/change-logs.ts

```diff
@@ -0,0 +1,24 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/**
+ * Represents the ChangeLog api object
+ */
+export interface ChangeLog {
+	id: number;
+	lastUpdated: Date;
+	level: string;
+	message: string;
+	ticketNum?: number | null;
+}
```

Comment:
This definition is provided by the `trafficops-types` library; any reason you made your own?

Reply:
Because the version we were using didn't have it.

Reply:
It does, you probably just didn't realize it. It's called `Log`, not `ChangeLog`, named after the API endpoint - which is `/logs` not e.g. `/change_logs`